### ISSUE TITLE
Setting hint programmatically results in no label 

### ISSUE THREAD

**Issue by sushant23** [rengwuxian/MaterialEditText#250]
As previously discussed in issue #37 when I set the hint programatically  it doesn't appear on floating label.

**Comment by sushant23** [comment]
I got around it by using methods setFloatingLabelText(hint) and setFloatingLabel(MaterialEditText.FLOATING_LABEL_NORMAL) in succession.

**Comment by berendn** [comment]
Indeed. When changing the hint in onCreateView the floating label keeps using the hint set in the XML instead of the setHint() method.
